romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -229,6 +236,12 @@ def ols_cas22(self, input_model, readnoise_model, gain_model):
         image_info = (slopes, ramp_dq, var_poisson, var_rnoise, err)
         image_model = create_image_model(input_model, image_info)
 
+        # Rescale by the gain back to DN/s
+        image_model.data /= gain[4:-4, 4:-4]
+        image_model.err /= gain[4:-4, 4:-4]
+        image_model.var_poisson /= gain[4:-4, 4:-4] ** 2
+        image_model.var_rnoise /= gain[4:-4, 4:-4] ** 2
+
```

Comment:
Staring at the schema, there is also var_flat, which we don't actually populate or do anything with.  I guess that's a separate ticket.

Reply:
This is actually fine and I was somehow missing the flat field uncertainty step computation later on.  Disregard this, sorry!